### DISCUSSION TITLE
我是秦始皇

### DISCUSSION THREAD

**Discussion by int-maxxx** [share121/inter-knot#1194] [其他]
vivo50看看实力

**Comment by lzxixizi** [comment]
我是荆轲
给皇上献上50城
你说这个匕首干啥用的？
这是给您切蛋糕用的

**Comment by Pasukalu** [comment]
我是秦始皇派蒙
![image](https://github.com/user-attachments/assets/b28e8861-8be6-47b2-bf02-c74d6b8794c8)
